app/assets/scripts/components/home/index.js

```diff
@@ -12,105 +21,369 @@ import {
+                        variation='base-raised-light'
+                        useIcon={['chevron-right--small', 'after']}
+                      >
+                        Learn more
+                      </Button>
+                    </StoryActions>
+                  </Story>
+                </IntroStories>
+              </IntroCopy>
+              {/*
+              <IntroMedia>
+              </IntroMedia>
+              */}
+              <MbMap
+                ref={this.mbMapRef}
+                onAction={this.onMapAction}
+                layers={[]}
+                activeLayers={[]}
+                // date={this.state.timelineDate}
+                aoiState={null}
+                comparing={false}
+              />
+              <Previous
+                as='a'
+                title='Previous story'
+                disabled={storyIndex === 0}
+                to='/'
+                variation='base-raised-light'
+                useIcon={['chevron-left--small', 'after']}
+                hideText
+                onClick={this.prevStory}
               />
-              <IntroActions>
-                <Button
-                  as={CleanNavLink}
-                  size='large'
-                  title='View the global map'
-                  to='/getstarted'
-                  variation='primary-raised-dark'
-                >
-                  Getting Started
-                </Button>
-                <Button
-                  as={CleanNavLink}
-                  size='large'
-                  title='View the Indicators'
-                  to='/indicators'
-                  variation='primary-raised-dark'
-                >
-                  Indicators
-                </Button>
-                <Button
-                  as={CleanNavLink}
-                  size='large'
-                  title='View the Spotlight locations'
-                  to='/spotlight'
-                  variation='primary-raised-dark'
-                >
-                  Spotlight locations
-                </Button>
-              </IntroActions>
-            </PageConstrainer>
+              <Next
+                as='a'
+                title='Previous story'
+                to='/'
+                disabled={storyIndex === stories.length - 1}
+                variation='base-raised-light'
+                useIcon={['chevron-right--small', 'after']}
+                hideText
+                onClick={this.nextStory}
+              />
+
+            </Intro>
           </InpageBody>
         </Inpage>
       </App>
     );
   }
 }
+
+Home.propTypes = {
+  fetchSpotlightSingle: T.func,
+  spotlight: T.object
+};
+
+function mapStateToProps (state, props) {
+  const spotlight = Object.keys(state.spotlight.single).reduce((accum, key) => {
+    return (
+      {
+        ...accum,
+        [key]: wrapApiResult(getFromState(state, ['spotlight', 'single', key]))
+      }
+    );
+  }, {});
+  return {
+    spotlight
+  };
```

Comment:
This should simplify things ;)
You can find an explanation around these redux helpers at https://github.com/NASA-IMPACT/covid-dashboard/blob/develop/app/assets/scripts/redux/reduxeed.md
```suggestion
  return {
    spotlight: wrapApiResult(state.spotlight.single, true)
  };
```

Reply:
ahh thank you for this haha